www/src/templates/template-starter-page.js

```diff
@@ -56,33 +56,13 @@ class StarterTemplate extends React.Component {
           maxWidth: 1080,
         }}
       >
+        <PageMetadata
+          title={`${repoName}: Gatsby Starter`}
+          image={screenshot.childImageSharp.fluid}
+          description={`Gatsby Starters: ${repoName}`}
+          timeToRead={1}
```

Comment:
Are the hardcoded "timeToRead" labels in pages like this useful? We may as well get rid of them.

Reply:
Actually... I'm trying to find information on `reading time` but I can't seem to find anything. Was this a feature that was taken out at some point?

Reply:
I've found information on `twitter:label1` only on third-party sites and it seems like it's been removed from the official [twitter developer docs](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/guides/getting-started). Playing around with the [validator](https://cards-dev.twitter.com/validator) doesn't show our time to read in any of our docs.

Reply:
for example there is a PR with tweak options for `timeToRead` and some code places where it comes from:

- #19763 `feat(gatsby-transformer-remark, gatsby-plugin-mdx) Make timeToRead configurable`

Reply:
`twitter:label1` is used in e.g. Slack for richer preview

![image](https://user-images.githubusercontent.com/16143594/75555388-6c50e580-5a3c-11ea-9198-0a60b3501cbe.png)

So for Slack `twitter:label1` and `twitter:label2` are valid and were added for this reason.

Reply:
As for your first question: I wouldn't say they are useful for starters and would switch the information for something like e.g. author or GitHub link.

Reply:
well that's not intuitive at all. I'm adding a comment :P

Reply:
I think switching the information is a good idea, but I'll leave that for a separate PR for now.